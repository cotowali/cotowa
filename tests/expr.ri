6 / 2 + 3 * 3
1 + 1 - 1
6 / 2 % 2
let v = 10
-v
+v

fn zero() int {
  let v = 0

  // evaluated but not used. this is valid but no output
  0
  v
  add(1, 2)

  return v
}

fn add(a int, b int) int {
  return a + b
}

zero()
add(3, 4)
