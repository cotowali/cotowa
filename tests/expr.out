12
1
1
-10
10
0
7
